tools/ab_test.py

```diff
@@ -285,7 +285,7 @@ def ab_performance_test(
             msg = (
                 f"\033[0;32m[Firecracker A/B-Test Runner]\033[0m A/B-testing shows a change of "
                 f"{format_with_reduced_unit(result.statistic, unit)}, or {result.statistic / old_mean:.2%}, "
-                f"(from {format_with_reduced_unit(old_mean, unit)} to {format_with_reduced_unit(new_mean, unit)}"
+                f"(from {format_with_reduced_unit(old_mean, unit)} to {format_with_reduced_unit(new_mean, unit)}) "
```

Comment:
maybe out of scope, but how about adding percentages in the output?

Reply:
That's what the commit for which this one is a "Fix" did :)

Reply:
ah, nice!